File: 1491. Average Salary Excluding the Minimum and Maximum Salary/Solution.cpp
class Solution {
public:
    double average(vector<int>& salary) {
        int total = 0, high = 0, low = 1e7;
        for (auto s : salary) {
            total += s;
            high = max(high, s);
            low = min(low, s);
        }
        return double(total - high - low) / (salary.size() - 2);
    }
};

File: 1491. Average Salary Excluding the Minimum and Maximum Salary/Solution.java
class Solution {
    public double average(int[] salary) {
        double total = 0, high = 0, low = 1e6;
        for (int s : salary) {
            total += s;
            high = Math.max(high, s);
            low = Math.min(low, s);
        }
        return (total - high - low) / (salary.length - 2);
    }
}